ci_framework/playbooks/07-admin-setup.yml

```diff
@@ -0,0 +1,18 @@
+- name: Run pre_admin_setup hooks
+  vars:
+    hooks: "{{ pre_operator_build | default([]) }}"
+    step: pre_admin_setup
```

Comment:
Care to mention that new hook keyword here ?
https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md#hooks